tests/test_atmos.py

```diff
@@ -7,7 +7,9 @@
 import xarray as xr
 
 from xclim import atmos, set_options
-from xclim.indices.helpers import make_hourly_temperature
+from xclim.core.units import convert_units_to
+from xclim.indices._conversion import shortwave_downwelling_radiation_from_clearness_index
```

Comment:
```suggestion
from xclim.indices import shortwave_downwelling_radiation_from_clearness_index
```

I realize that this shouldn't matter since we control the API, but we can probably avoid linking to the internal API.